recipes/microhapdb/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.2" %}
+{% set sha256 = "450faabc3420fc7ed9107c0de3a0643b82352a592bde487c4afc6dc064bc8f7a" %}
+
+package:
+  name: microhapdb
+  version: '{{version}}'
+
+source:
+  url: https://github.com/bioforensics/microhapdb/archive/{{ version }}.tar.gz
+  sha256: '{{sha256}}'
+
+build:
+  noarch: python
+  entry_points:
+    - microhapdb = microhapdb.cli:main
+  script: python -m pip install --no-deps --ignore-installed .
+  number: 1
```

Comment:
You should start at 0 for new versions

Reply:
Thanks for the tip. I cobbled this together by looking at a few examples, and I'm still trying to work out the details. :-)